decidim-admin/app/models/decidim/admin/abilities/participatory_process_collaborator_user.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    module Abilities
+      # Defines the abilities for a collaborator user in the admin
+      # section. Intended to be used with `cancancan`.
+      class ParticipatoryProcessCollaboratorUser < Decidim::Abilities::ParticipatoryProcessCollaboratorUser
```

Comment:
Why appending `User` at the end of abilities? If we had to append anything, shouldn't it be `Ability`?